recipes/conda-libmamba-solver/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "conda-libmamba-solver" %}
+{% set version = "22.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/conda-incubator/conda-libmamba-solver/archive/{{ version }}.tar.gz  # temporary
+  # sha256: 2b3a0c466fb4a1014ea131c2b8ea7c519f9278eba73d6fcb361b7bdb4fd494e9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - flit-core >=3.2,<4
+  run:
+    - python >=3.6
+    - libmambapy >=0.22
+    - conda >=4.12
```

Comment:
```suggestion
    - conda >=4.12.0
```

Reply:
This is equivalent, I'd swear.

Reply:
Yeah, true, and yet, I can't unsee SemVer, I know it's all a lie :P